tests/Behat/MediaContext.php

```diff
@@ -82,6 +108,176 @@ public function visitMediaPage(string $name): void {
     $this->visitPath($media->toUrl()->toString());
   }
 
+  /**
+   * Creates media documents with the specified file names.
+   *
+   * Usage example:
+   *
+   * Given the following documents:
+   *   | name   | file   |
+   *   | name 1 | file 1 |
+   *   |   ...  |   ...  |
+   *
+   * @Given the following documents:
```

Comment:
```suggestion
   * @Given the following document(s):
```
would be nice to have